wire/.editorconfig

```diff
@@ -0,0 +1,31 @@
+; This file is for unifying the coding style for different editors and IDEs.
+; More information at http://editorconfig.org
+
+
+root = true
+
+
+[*]
+charset = utf-8
+end_of_line = lf
+
+
+[*.{php,inc,module}]
+tab_width = 4
```

Comment:
Do we need to specify the tab_width? I mean if it's tabs people should in theory be able to use the tab width they like.